engine/ingame/view.py

```diff
@@ -97,3 +103,13 @@ def _center_camera_to_player(self) -> None:
         player_centered = screen_center_x, screen_center_y
 
         self.camera.move_to(player_centered)
+
+    def on_resize(self) -> None:
```

Comment:
Consider renaming this, calling it `on_resize` is confusing because:
1) Usually methods named `on_*` are event handlers, and an `on_resize` is typically an event handler for when the window is resized.
2) It doesn't describe what the function does. With the exception of event handlers, the name of a function should typically describe what it does.

However I would even take a step back - the camera position centering is independent in two axes, but the way this is getting factored is if at least one axis needs centering. You'll naturally end up duplicating code if one axis doesn't need to be centered.
Instead what I'd do is put the logic into `_center_camera_to_player()` and handle each axis separately.